src/ui/screens/AddAccountScreen/AddressPreview.tsx

```diff
@@ -53,27 +55,29 @@ function AddressInfoPreview(props: PropTypes) {
               right={() => <Caption>{accountInfo.balance?.formattedFree}</Caption>}
             />
           )}
-          <List.Item
-            title="Judgment"
-            left={() => (
-              <View style={globalStyles.justifyCenter}>
-                <Icon name="hammer" size={20} />
-              </View>
-            )}
-            right={() =>
-              accountInfo?.registration?.judgements?.map((judgment, i) => {
-                if (judgment) {
-                  return (
-                    <JudgmentStatus
-                      key={i}
-                      registrationJudgement={judgment}
-                      hasParent={Boolean(accountInfo?.registration?.displayParent)}
-                    />
-                  );
-                }
-              })
-            }
-          />
+          {accountInfo?.registration?.judgements && accountInfo?.registration?.judgements.length > 0 ? (
```

Comment:
Instead of completely removing let's give the information `No judgements provided`